api/src/main/java/org/apache/cloudstack/api/command/admin/offering/CreateServiceOfferingCmd.java

```diff
@@ -56,7 +56,7 @@ public class CreateServiceOfferingCmd extends BaseCmd {
     @Parameter(name = ApiConstants.CPU_NUMBER, type = CommandType.INTEGER, required = false, description = "the CPU number of the service offering")
     private Integer cpuNumber;
 
-    @Parameter(name = ApiConstants.CPU_SPEED, type = CommandType.INTEGER, required = false, description = "the CPU speed of the service offering in MHz.")
+    @Parameter(name = ApiConstants.CPU_SPEED, type = CommandType.INTEGER, required = false, description = "the CPU speed of the service offering in MHz. In the case cgroups are used on KVM, this is interpreted to calculate the number of shares. see also https://libvirt.org/formatdomain.html#cpu-tuning")
```

Comment:
@DaanHoogland, I think it would benefit the users if we can add more detail on this parameter description. Something like below, what do you think?
```suggestion
    @Parameter(name = ApiConstants.CPU_SPEED, type = CommandType.INTEGER, required = false, description = "the CPU speed of the service offering in MHz. For the KVM hypervisor," +
            " the values of the parameters cpuSpeed and cpuNumber will be used to calculate the `shares` value. This value is used by the KVM hypervisor to calculate how much time" +
            " the VM will have access to the host's CPU. The `shares` value does not have a unit, and its purpose is be a weight value for the host to compare between its guest" +
            " VMs. For more information, see https://libvirt.org/formatdomain.html#cpu-tuning.")
```

Reply:
I updated your suggestion @BryanMLima  (and I was really surprised that I was allowed to???)

I added the more explicit mention of VMware and Xen. Hope you don't mind.